ballet/validation/entropy.py

```diff
@@ -19,12 +19,14 @@ def calculate_disc_entropy(X):
     take datasets of mixed discrete and continuous functions.
 
     Args:
-        X (array-like): An array with shape (n_samples, n_features)
+        X (array-like): An array-like (np arr, pandas df, etc.) with shape
```

Comment:
for readability, please use `numpy array` or `np array`. (here and the other docstrings)